meta/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Role for https://github.com/hzeller/gmrender-resurrect
```

Comment:
I would explain what kind of software it is, not where to find it

Reply:
They have a readme there, explaining what software this is. They do this in more detail an with more competence, than I could do here. An in-place explanation therefore wouldn't replace the need for a link. If at all, it would be additional.